api/src/main/resources/application-cloud.properties

```diff
@@ -16,3 +16,4 @@
 #
 
 spring.flyway.ignore-missing-migrations=true
+slack.url=""
```

Comment:
If this property is set in application.properties, you don't need to include it in other profiles unless you want to change its value